test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -25,7 +25,7 @@ notifyFormat_ngsi10
 
 --SHELL-INIT--
 dbInit CB
-brokerStart CB 38,101 IPv4 -statSemWait
+brokerStart CB 38,101 IPv4 -statSemWait --cache
```

Comment:
Ok, by the moment and while we can address a better refactoring in the brokerStart function (btw, which number has that issue?) the criteria of `-` for "options to be passed to CB" and `--` for "options to be consumed by the brokerStart() function seems ok.

Regarding:

> The best option actually would be to modify the broker to use '--' instead of '-' for the long options. Support both '-' and '--', to finally drop '-'.

That involves a very deep change which would break all the tooling which relies in CB parametrization (Puppet, Ansible, etc.) with very little gain. Thus, I think is better to keep as CLI works now.